### PR TITLE
feat: CHK-2416 add GET wallet in payment flow for io (eCommerce for IO API)

### PR DESCRIPTION
**Pull request by infantesimone**
### List of changes

The goal of this PR is to add `GET wallets` in eCommerce for IO API according to the following [RFC](https://pagopa.atlassian.net/wiki/spaces/I/pages/910688286/RFC+-+sessioni+tra+eCommerce+e+wallet).
The new openapi is https://editor.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pagopa-infra/5ab08f356d4697328de95079b1e2225e97932d51/src/domains/ecommerce-app/api/ecommerce-io/v1/_openapi.json.tpl

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_get_wallets_by_user.xml.tpl [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<policies>
```
Comment (pietro-tota) [inline src/domains/ecommerce-app/api/ecommerce-io/v1/_get_wallets_by_user.xml.tpl:1]: Is this policy needed? it just call wallet service and return it's response body as is for ok responses. maybe it just be removed and let base policy handle wallet b.e. call

**Comment by infantesimone** [comment]
@mastro993 @forrest57 @Hantex9  feedback for the new openapi api with GET Wallet in the payment flow?
[https://raw.githubusercontent.com/pagopa/pagopa-infra/5ab08f356d4697328de95079b1e2225e97932d51/src/domains/ecommerce-app/api/ecommerce-io/v1/_openapi.json.tpl](https://editor.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pagopa-infra/5ab08f356d4697328de95079b1e2225e97932d51/src/domains/ecommerce-app/api/ecommerce-io/v1/_openapi.json.tpl)

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_get_wallets_by_user.xml.tpl (hunk 1 above)
Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-io/v1/_get_wallets_by_user.xml.tpl:1]: ah ok my bad :D

**Review by pietro-tota — APPROVED** [review]

**Review by pietro-tota — APPROVED** [review]
🚀

**Pull request merged**